content/en/posts/css/introduction-to-cssnext/index.md

```diff
@@ -0,0 +1,368 @@
+---
+date: "2015-XX-XX"
+title: Introduction to cssnext
+tags:
+  - css
+  - postcss
+authors:
+  - Macxim
+header:
+  credit: https://www.flickr.com/photos/frinky/620935482
+---
+
+From [the official website](https://cssnext.github.io/):
+> **cssnext** is a CSS transpiler that allows you to use tomorrow's CSS syntax today. It transforms CSS specs that are not yet implemented in popular browsers into more compatible CSS.
+
+## What does it mean?
+
+Since CSS3, you may have heard that CSS is divided into several independent documents called "modules". Those modules can have different levels of stability and [statuses](http://www.w3.org/Style/CSS/current-work#legend). It can take quite some time for the browsers to implement those modules and even more for the W3C to give them the ultimate status of [Recommandation](http://www.w3.org/2005/10/Process-20051014/tr#RecsW3C).
+
+With **cssnext**, you can use the syntax of [CSS Module Level 4](http://www.xanthir.com/b4Ko0) such as _custom properties_ or _custom media queries_. **cssnext** will transform this new strange syntax into something that the browser can actually understand.
+
+In short, it makes you **taste the future**.
+
+I don't know about you but speaking as someone who loves to experiment with cutting-edge technologies, I think this is pretty cool!
+
+You should take a look at the [feature list](https://github.com/cssnext/cssnext/blob/master/README.md#features).
```

Comment:
Point to the website here, this link will be deprecated when I will merged website branch